### PR TITLE
204 better contributing.md

### LINKED ISSUE #204: [doc] better contributing.md
- [x] some link doesn't work or are in double
- [x] make pr look like issue section
- [ ] add legend on png
- [x] fix typo
- [x] some other thing

### PR DESCRIPTION
**Pull request by Draeggon**
fix #204
improve contributing.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,49 +38,54 @@ You need to fork [scratchy repository](https://github.com/mdl29/scratchy)
...
+        - the id of the issue resolved with a keyword to link to the coresponding issues
+          > ℹ️ [how to link a issue and use the keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)(Fixes,resolves...)
+        - a message who say what you add or do in your pull request.
 
-    Add [labels](https://github.com/mdl29/scratchy/labels) (Frontend,Backend,documentation...), add to project your pull request(V1,V2) and milestone depending your issue.
-    
-* **Real Example :**   
-   * if you want to solve #91 issue you should have named your pull request : `resolves #91 : Create a scratchy service` 
-  ![example-issue](./doc/example/example_pr_title.png)
+3. Assign it to yourself
```
Comment (Benvii) [inline CONTRIBUTING.md:75]: What if the reporter doesn't know how to solve it ^^

**Review by Benvii — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Draeggon) [reply CONTRIBUTING.md:75]: oops

Comment (Draeggon) [reply CONTRIBUTING.md:75]: wait it is a pull request so if you write it you are the one that solved the problem

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -40,51 +38,55 @@ You need to fork [scratchy repository](https://github.com/mdl29/scratchy)
...
 ## **Create a pull request :**  
 
 When you think your code is ready to be merged to the project create a pull request.
 
-1. choose the title:
-    when your pull request is ready link to an issue :
-
-    * **example :**` Resolves #ISSUEID : title of the issue `
+1. Choose the title:
+    patern: `ISSUEID title of the issue `
```
Comment (yannis-mlgrn) [inline CONTRIBUTING.md:63]: I think is better `Resolves #ISSUEID : title of the issue` or a title like this.
It's more design xd

**Review by yannis-mlgrn — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (Draeggon) [reply CONTRIBUTING.md:63]: just putting resolves in front of every pr don't add anything
i prefer to stick with something closer to the [git convention](https://namingconvention.org/git/pull-request-naming.html)

Comment (yannis-mlgrn) [reply CONTRIBUTING.md:63]: hmm i think is needed to request @Benvii to choose the format

**Comment by yannis-mlgrn** [comment]
else all seem right good jobs

File: CONTRIBUTING.md (hunk 2 above)
Comment (js-all) [reply CONTRIBUTING.md:63]: I propose my own, self-made convention: the DWTFYWALAIRAUNOGASA(EY)
Do
Whatever
The
Fuck
You
Want
As
Long
As
It's
Readable
And
Understandable,
No
One
Gives
A
Shit
Anyways
(
Except
You
)

Comment (Draeggon) [reply CONTRIBUTING.md:63]: i think you could improve the name but it seem to be an interesting proposition
could you make a linter to help people use it :joy:

**Review by Benvii — APPROVED** [review]

**Review by yannis-mlgrn — APPROVED** [review]

**Pull request merged**